packages/core/src/boot/init.ts

```diff
@@ -44,6 +44,8 @@ export const INTAKE_SITE = {
   [Datacenter.US]: 'datadoghq.com',
 }
 
+export const NEW_INTAKE_DOMAIN_ALLOWED_SITES = [INTAKE_SITE[Datacenter.US], 'datad0g.com']
```

Comment:
Since it will now be enabled by customer, I'd be in favor of supporting all sites out of the box, in order to not have to updates the sdk when the intakes will be ready

Reply:
Makes sense. But if the user uses `useAlternateIntakeDomains: true` while the alternate intake for their site is not ready yet, we will not be able to fallback to the classic intake. is that ok?

Reply:
I would say yes, we could specify in the doc in which data center it is available and only update the doc when a new domain is up